### PR TITLE
As/preference

### PR DESCRIPTION
**Pull request by Apsaliya**
Preferences (Just UI) + Rate Us + Analytics

### CODE REVIEW

**Comment by Apsaliya** [comment]
CI is failing because play services json is missing. Hmmmmm, will have to think about it.

**Comment by Apsaliya** [comment]
We'll add probably template of that json which will hopefully make everyone happy including CI. Anyway, @Hirak20 You can go ahead with review.

File: app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      }
+
+      btYes.setOnClickListener {
+        onYesSelected(activity)
+        try {
+          val viewIntent = Intent(Intent.ACTION_VIEW,
+              Uri.parse("https://play.google.com/store/apps/details?id=wordtextcounter.details.main"))
+          activity.startActivity(viewIntent)
+        } catch (e: Exception) {
+          Toast.makeText(activity, "Oops! We were not able to reach play store. Sorry for that!", Toast.LENGTH_LONG).show()
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt]: move to strings.xml

File: app/src/main/java/wordtextcounter/details/main/analytics/AnalyticsHelper.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  fun disableAnalytics() {
+    analyticsInstance?.setAnalyticsCollectionEnabled(false)
+    analyticsInstance = null
+  }
+
+  fun enableAnalytics(context: Context) {
+    if (analyticsInstance == null) {
+      analyticsInstance = FirebaseAnalytics.getInstance(context)
+    }
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/analytics/AnalyticsHelper.kt]: If it's nonnull, need to setAnalyticsCollectionEnabled to true here

**Comment by Apsaliya** [comment]
Only thing that remains for this now is back press handling when rate us dialog is showed. I'll do that as a separate task once we have dialog UI (with all the actions) ready.

**Pull request merged**